internal/cmd/grants.go

```diff
@@ -97,17 +97,19 @@ func newGrantRemoveCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "remove IDENTITY DESTINATION",
 		Aliases: []string{"rm"},
-		Short:   "Revoke access to a destination",
-		Long: `Revokes access that user has to the destination.
+		Short:   "Revoke access from a destination",
+		Example: `# Remove all grants of an identity in a destination
+$ infra grants remove janedoe@example.com kubernetes.docker-desktop 
+$ infra grants remove machineA kubernetes.docker-desktop
 
-IDENTITY is one that was being given access.
-DESTINATION is what the identity will lose access to. 
```

Comment:
This is still important information the user should have. It describes what the command does rather than how to use it. There's no reason not that have both a long description and examples

Reply:
Would add a mention of identity to the description vs outlining the arguments (examples do this well)

i.e. `Revoke an identity's access to a destination`